web/packages/api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snowbridge/api",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
With https://github.com/Snowfork/snowbridge/pull/1535, we can now publish packages or trigger the release flow without manually updating the version number.

See https://github.com/Snowfork/snowbridge/actions/runs/17366136623 which auto-released and bumped `@snowbridge/api` to [0.2.2](https://github.com/Snowfork/snowbridge/releases/tag/%40snowbridge%2Fapi%400.2.2.).

So there is no need to manually edit the version number here. If we do a manual release, we should bump the version to 0.2.3.